OpenRA.Game/Input/IInputHandler.cs

```diff
@@ -61,6 +62,33 @@ public enum Modifiers
 		Meta = 8,
 	}
 
+	public static class ModifiersExts
+	{
+		[TranslationReference]
+		const string Cmd = "keycode-modifier.cmd";
+
+		[TranslationReference(Traits.LintDictionaryReference.Values)]
+		static readonly Dictionary<Modifiers, string> ModifierTranslationKeys = new()
+		{
+			{ Modifiers.None, "keycode-modifier.none" },
+			{ Modifiers.Shift, "keycode-modifier.shift" },
+			{ Modifiers.Alt, "keycode-modifier.alt" },
+			{ Modifiers.Ctrl, "keycode-modifier.ctrl" },
+			{ Modifiers.Meta, "keycode-modifier.meta" },
+		};
+
+		public static string DisplayString(Modifiers m)
+		{
+			if (m == Modifiers.Meta && Platform.CurrentPlatform == PlatformType.OSX)
```

Comment:
```suggestion
			if (m == Modifiers.Meta)
```

this shouldn't be possible on non-OSX, but if it were/is then we would fail to provide a translation.

Reply:
I don't follow - on non OSX we'd use the translation for Meta .

Reply:
Ah, I thought the translation for meta was always the cmd one, but if there is a different one then never mind.